gitpod-io/gitpod
# [dev] Move BPF VM creation into image build

Fixes #4945.

Also, makes the `img` even smaller by changing the format from `raw` to `qcow2`.
![image](https://user-images.githubusercontent.com/32448529/127017856-2d25c77b-f9a6-450d-808c-f75b9d3c2eb9.png)

### Test
 - start a workspace on this branch
 - see that the VM is still started up on workspace start, but without the init phase/log


### Future work:
 - build time is still quite high if we need to rebuild the `dev` image. Converging all in-VM steps into a single script instead of starting/stopping it for every step should help.
